src/pages/Diet/Diet.tsx

```diff
@@ -48,15 +54,66 @@ const getWeek = (day: Dayjs): Array<{ date: Dayjs; dayname: string }> => {
   return week;
 };
 
-export const Diet = (props: Props) => {
+export const Diet = () => {
   const [config, setConfig] = useState<CalendarConfigType>(calendarConfig);
-  const [isLoaded, setIsLoaded] = useState(false);
+  const [selectedClientId, setSelectedClientId] = useState<string>('');
   const { isMobile } = useContext(AppContext);
+  const { role } = useContext(AppContext);
 
   function onReserve(itemId: string, tileId: string) {
     console.log(itemId, tileId);
   }
 
+  function onColumnReserve(itemId: string, columnId: string) {
+    console.log(itemId, columnId);
+  }
+
+  async function onClientChange(event: SelectChangeEvent) {
+    console.log(event.target.value);
+    setSelectedClientId(event.target.value);
+  }
+
+  const { data: dietDayTemplates } = useQuery(
+    ['diet-day-templates'],
+    async () => {
+      const { data } = await axios.get<Array<{ name: string; id: string }>>(
+        'http://localhost:8081/api/dietDay/template/truncated',
+        {
+          headers: { Authorization: `Bearer ${getCookie('userToken')}` },
+        }
+      );
+      return data;
+    },
+    { enabled: false }
+  );
+  console.log('dietDayTemplates: ', dietDayTemplates);
+
+  const { data: coachClients } = useQuery(['coach-clients'], async () => {
+    const { data } = await axios.get<Array<UserProps>>(
+      'http://localhost:8081/api/user/coach/clients',
+      {
+        headers: { Authorization: `Bearer ${getCookie('userToken')}` },
+      }
+    );
+    return data;
+  });
```

Comment:
Albo z mojej PR można zrobić `invalidateQueries` albo ja to przerobię na swojej PR żeby nie powielać tego samego query